README.md

```diff
@@ -427,6 +433,30 @@ const uiSchema = {
 };
 ```
 
+#### `addable` option
+
+if either `items` or `additionalItems` contains a schema object, an add button for new items is shown by default. You can turn this off with the `addable` option in `uiSchema`:
```

Comment:
I feel like this is trying to fix a misused json schema; If you want a fixed size array, your schema should reflect that intent, and the lib should render the form appropriately. Have a look at the `fixedItemsList` sample in the playground.

I understand that this would add more flexibility, but on the other hand I feel like providing features to help people fixing their schemas is an endless rabbit hole we're really wanting to start exploring... Thoughts? Did you know about support for fixed size arrays?